simulations/vip-512/unichainsepolia.ts

```diff
@@ -0,0 +1,181 @@
+        });
+
+        it(`should set reserve factor to 0.2`, async () => {
+          expect(await vToken.reserveFactorMantissa()).to.equal(WBTCMarket.riskParameters.reserveFactor);
+        });
+
+        it(`should set collateral factor to 0.77`, async () => {
+          const market = await comptroller.markets(WBTCMarket.vToken.address);
+          expect(market.collateralFactorMantissa).to.equal(WBTCMarket.riskParameters.collateralFactor);
+        });
+
+        it(`should set liquidation threshold to 0.8`, async () => {
+          const market = await comptroller.markets(WBTCMarket.vToken.address);
+          expect(market.liquidationThresholdMantissa).to.equal(WBTCMarket.riskParameters.liquidationThreshold);
+        });
+
+        it(`should set protocol seize share to 0.05`, async () => {
+          expect(await vToken.protocolSeizeShareMantissa()).to.equal(parseUnits("0.05", 18));
+        });
+
+        it(`should set supply cap to 400`, async () => {
+          expect(await comptroller.supplyCaps(WBTCMarket.vToken.address)).to.equal(WBTCMarket.riskParameters.supplyCap);
+        });
+
+        it(`should set borrow cap to 40`, async () => {
+          expect(await comptroller.borrowCaps(WBTCMarket.vToken.address)).to.equal(WBTCMarket.riskParameters.borrowCap);
+        });
+      });
+
+      describe("risk parameters for USDTO", () => {
+        before(async () => {
+          vToken = await ethers.getContractAt(VTOKEN_ABI, USDTOMarket.vToken.address);
+          comptroller = await ethers.getContractAt(COMPTROLLER_ABI, COMPTROLLER_CORE);
+        });
+
+        it(`should set reserve factor to 0.1`, async () => {
+          expect(await vToken.reserveFactorMantissa()).to.equal(USDTOMarket.riskParameters.reserveFactor);
+        });
+
+        it(`should set collateral factor to 0.77`, async () => {
+          const market = await comptroller.markets(USDTOMarket.vToken.address);
+          expect(market.collateralFactorMantissa).to.equal(USDTOMarket.riskParameters.collateralFactor);
+        });
+
+        it(`should set liquidation threshold to 0.8`, async () => {
+          const market = await comptroller.markets(USDTOMarket.vToken.address);
+          expect(market.liquidationThresholdMantissa).to.equal(USDTOMarket.riskParameters.liquidationThreshold);
+        });
+
+        it(`should set protocol seize share to 0.05`, async () => {
+          expect(await vToken.protocolSeizeShareMantissa()).to.equal(parseUnits("0.05", 18));
+        });
+
+        it(`should set supply cap to 50000000`, async () => {
+          expect(await comptroller.supplyCaps(USDTOMarket.vToken.address)).to.equal(
+            USDTOMarket.riskParameters.supplyCap,
+          );
+        });
+
+        it(`should set borrow cap to 45000000`, async () => {
+          expect(await comptroller.borrowCaps(USDTOMarket.vToken.address)).to.equal(
+            USDTOMarket.riskParameters.borrowCap,
+          );
+        });
+      });
+    });
+
```

Comment:
We could add checks for the prices, similar to the checks we added for unichainmainnet